docs/readme.md

```diff
@@ -30,54 +30,135 @@ https://user-images.githubusercontent.com/1251986/154977573-4015e77c-43e7-481e-a
 
 ---
 
-### Stack
+### Build your custom storefront
 
-- Typescript
-- React
-- Nextjs
-- GraphQL
-- Mui
-- Apollo Client
-- GraphQL Code Generator
-- Framer Motion
```

Comment:
We're removing this here, should we find another place, this is one of the things mentioned by customers that they were looking for this stack.

Reply:
Yes this will be moved to another location